README.md

```diff
@@ -189,11 +189,47 @@ $ git diff --word-diff
 <dt>How do I force a line break?</dt>
 <dd>
 
-You can add a hard line break with the `<br/>` element.
-Although CommonMark and other lightweight markup languages
-allow trailing spaces to indicate breaks between consecutive lines,
-this syntax is incompatible with
-editors that automatically strip trailing whitespace.
+Some lightweight markup languages provide a dedicated syntax for line breaks,
+some support line breaks using the inline HTML element `<br>`\*,
+and some do both.
+
+<dl>
+
+<dt>AsciiDoc</dt>
+<dd>**Syntax**: A trailing space followed by a plus sign (`+`)</dd>
+<dd>**Inline HTML**: Not supported</dd>
+
+<dt>CommonMark</dt>
+<dd>**Syntax**: A trailing backslash (`\`) \*\*</dd>
+<dd>**Inline HTML**: Supported</dd>
+
+<dt>Markdown</dt>
+<dd>**Syntax**: None</dd>
+<dd>**Inline HTML**: Supported</dd>
+
+<dt>MediaWiki</dt>
+<dd>**Syntax**: None</dd>
+<dd>**Inline HTML**: Supported</dd>
+
+<dt>OrgMode</dt>
+<dd>**Syntax**: Two trailing backslashes (`\\`)</dd>
+<dd>**Inline HTML**: Not supported</dd>
+
+<dt>reStructuredText</dt>
+<dd>**Syntax**: Leading vertical line (`|`) in a [line block][rst-line-block]</dd>
+<dd>**Inline HTML**: Not supported</dd>
+
+> \*
+> The _Line Break_ element is represented as
+> an empty tag (`<br>`) in HTML,
+> and either a tag pair (`<br></br>`) or
+> a self-closing tag (`<br />`) in XHTML.
+
+> \*\*
+> Although CommonMark allows the use of trailing spaces
+> to indicate breaks between consecutive lines,
+> this syntax is incompatible with
+> tools that automatically strip trailing whitespace.
```

Comment:
Other drawbacks of trailing spaces:

- Whitespace is not visible in all editors, so (accidental) trailing spaces can cause confusion.
- Trailing spaces can cause unnecessary diffs if some developers' editors automatically strip trailing whitespace while some other developers' editors do not.

Not sure if it's necessary or relevant to mention these in the spec.